labtool2.0/src/tests/__snapshots__/Courses.test.js.snap

```diff
@@ -99,6 +99,9 @@ exports[`<Courses /> Courses Component should render correctly 1`] = `
                   to="/labtool/courses/TKT20010.2018.K.A.1"
                 >
                   Aineopintojen harjoitustyö: Tietorakenteet ja algoritmit
+                   (
+                  17-18 4.period
```

Comment:
Should this and the rest be in Finnish?

Reply:
Is there any difference if the period is shown in English or Finnish?

Reply:
Perhaps not. I was thinking merely for consistency.